scripts/build-win.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs')
+
+const dir = './apks'
+
+
+async function copyAPK() {
+  if (!fs.existsSync(dir)){
```

Comment:
Use https://nodejs.org/api/fs.html#fspromisesaccesspath-mode with constants.R_OK to check if a file exists asynchronously